markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Editing freesewing content on Github
+---
+
+## Introduction
+This 'How to' is designed to help get you started editing content using the [Github website](https://github.com/freesewing).
+
+Github is a website that provides storage for all the Freesewing code and content.  It allows our supporters to make suggested edits and helps the administrators to review and implement approved changes. 
+
+If you are new to Github, you will need to [create an account](https://github.com/signup?user_email=&source=form-home-signup).Github For info, it is also possible to make changes using the Git desktop application and/or your computer's command line, but these advanced topics are beyond the scope of this 'How to'.
```

Comment:
There are one or two typos here, around ".Github For info, it is".
- There should be a space after the period character.
- Should the start of the second sentence be changed simply to "It is"? (I am not sure what was intended.)

Also, it should be "GitHub Desktop application" instead of "Git desktop application". (The official name of the app is "GitHub Desktop", per https://desktop.github.com .

Also, GitHub seems to style their name as "GitHub", with a capital "H". I am not sure whether or not it matters how we style the word in this document, but I wanted to bring it up.

Reply:
Thanks for these.  Some are typos I had already fixed (in a lost branch *rolls eyes).  I think the phrase "GitHub desktop application" makes sense in context, as newbies might not know what "GitHub desktop" is, without the word "application."
I have changed Github to GitHub throughout - good spot!